src/pages/Dashboard/Markets/SupplyMarket/SuppliedTable.tsx

```diff
@@ -0,0 +1,89 @@
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import { formatCoinsToReadableValue, formatApy } from 'utilities/common';
+import { Asset, TokenId } from 'types';
+import { switchAriaLabel, Token, Toggle } from 'components';
+import { Table, ITableProps } from 'components/v2/Table';
+import { useTranslation } from 'translation';
+
+export interface ISuppliedTableUiProps {
+  assets: Asset[];
+  isXvsEnabled: boolean;
+  setSelectedAsset: (asset: Asset | undefined) => void;
+  collateralOnChange: (asset: Asset) => void;
+}
+
+export const SuppliedTable: React.FC<ISuppliedTableUiProps> = ({
+  assets,
+  isXvsEnabled,
+  collateralOnChange,
+  setSelectedAsset,
+}) => {
+  const { t } = useTranslation();
+
+  const columns = useMemo(
+    () => [
+      { key: 'asset', label: t('markets.columns.asset'), orderable: false },
+      { key: 'apyEarned', label: t('markets.columns.apyEarned'), orderable: true },
+      { key: 'balance', label: t('markets.columns.balance'), orderable: true },
+      { key: 'collateral', label: t('markets.columns.collateral'), orderable: true },
+    ],
+    [],
+  );
+  // Format assets to rows
+  const rows: ITableProps['data'] = assets.map(asset => [
+    {
+      key: 'asset',
+      render: () => <Token symbol={asset.symbol as TokenId} />,
+      value: asset.id,
+    },
+    {
+      key: 'apyEarned',
+      render: () => {
+        const apy = isXvsEnabled ? asset.xvsSupplyApy.plus(asset.supplyApy) : asset.supplyApy;
+        return formatApy(apy);
+      },
+      value: asset.supplyApy.toString(),
+    },
+    {
+      key: 'balance',
+      render: () =>
+        formatCoinsToReadableValue({
+          value: asset.supplyBalance,
+          tokenId: asset.symbol as TokenId,
+        }),
```

Comment:
This should display two values:
<img width="172" alt="Screen Shot 2022-04-26 at 22 11 05" src="https://user-images.githubusercontent.com/44675210/165393159-99eca0d2-797a-485b-89ba-b68c04579691.png">